### ISSUE TITLE
Add LinkButton

### ISSUE THREAD

**Issue by vipentti** [debois/elm-mdl#201]
This recently came up with trying to implement https://vipentti.github.io/elm-mdl-dashboard/ 's `View Source` button. 

In the actual MDL template the `View Source` button is technically Material button but it is a link-element (`a`).

Now I am uncertain if there is an actual need for this as you can somewhat work around this issue (opening a link) either by using Ports (required if you need to open the link in a new tab using window.open to emulate `target = "_blank"`) or for example `Navigation.newUrl`.

Basically this would be a `Button` in all ways except the `button` -element would be swapped to `a` and some additional attributes added: `href` and `target` at least.

**NOTE** This not strictly specified in either MDL and Material Spec. However, MDL allows this as it uses CSS classes and Material Spec does not take into account the type of the element, as long as it is a button which this technically still is, only the Html element is changed but the functionality is not.

**Comment by debois** [comment]
(Misclick, sorry.)
